### PR TITLE
BT wifi pilot docs

### PR DESCRIPTION
**Pull request by benilovj**
### Context

We need to host the privacy notice, user guide and provide a link to support

### Changes proposed in this pull request

image

image

### CODE REVIEW

File: app/views/pages/bt_wifi_privacy_notice.html.erb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+<!--
+
+
+#
+
+
+For further information and how to request your data, please use the ‘contact form’ in the Personal Information Charter at
+
+https://www.gov.uk/government/organisations/department-for-education/about/personal-informa1tion-charter
```
Comment (aldavidson) [inline app/views/pages/bt_wifi_privacy_notice.html.erb:78]: is there a typo in `personal-informa1tion-charter` ?

Comment (aldavidson) [reply app/views/pages/bt_wifi_privacy_notice.html.erb:78]: I guess it's a comment, so not a blocker

Comment (benilovj) [reply app/views/pages/bt_wifi_privacy_notice.html.erb]: oops 😬  – that whole thing needs to go. fixed and force pushed

**Comment by fofr** [comment]
This uses the old BT user guide.

**Pull request merged**